README.md

```diff
@@ -185,19 +185,20 @@ Here is a summary of each parameter:
        e. packed
        f. csvfiles[]
        g. json_plotfile
-       h. json_args
-       i. update_interval
-       j. history_length
-       k. raw_data_sources[]
-       l. threshold
-       m. sort_datasets
-       n. x_min
-       o. x_max
-       p. y_min
-       q. y_max
-       r. x_log_scale
-       s. y_log_scale
-       t. scatterplot
+       h. json_object
```

Comment:
It appears this list all merges up in to one paragraph.  Is that intended?

Reply:
I believe it was already in one paragraph but would be better if displayed in a list. I'm going to improve the formatting throughout the documentation since this was originally a .txt file.